src/System.Net.Sockets/src/System.Net.Sockets.csproj

```diff
@@ -441,7 +431,15 @@
       <Link>Interop\Unix\System.Native\Interop.Write.cs</Link>
     </Compile>
   </ItemGroup>
-  <ItemGroup Condition="'$(TargetGroup)' == 'net46'">
+  <ItemGroup Condition="'$(TargetGroup)'=='netcore50'">
+    <Compile Include="$(CommonPath)\Microsoft\Win32\SafeHandles\SafeHandleZeroOrMinusOneIsInvalid.cs">
+      <Link>Common\Microsoft\Win32\SafeHandles\SafeHandleZeroOrMinusOneIsInvalid.cs</Link>
+    </Compile>
```

Comment:
So, are we changing our source code where we use "if netcore50". And we use that term in our folder names and partial class names.  The documentation needs to be updated if we don't use "netcore50" TFM anymore.  We have separate implementations for some of the networking libraries, so need good guidelines to keep this organized and building correctly.